apps/desktop-wallet/src/api/addressesTokensPricesDataHooks.ts

```diff
@@ -52,7 +55,7 @@ export const useAddressesTokensPrices = () => {
               symbol: symbols[i]
             }) as TokenPrice
         ),
-      initialData: () =>
+      placeholderData: () =>
```

Comment:
One big difference between `initialData` and `placeholderData` is that the former is added to the cache and the query is considered `fresh` until it becomes `stale` based on the `staleTime`, whereas the latter is not added to the cache and the query runs immediately. That's the behavior we want here.